openshift/templates/jobs/backups/README.md

```diff
@@ -0,0 +1,111 @@
+# How to Configure Mongo Database Backup Cronjob and Restore From it
+
+The OpenShift template that we are currently using to run our mongo database backups is ```backup-cron.yaml``` in this directory.
+This is based on the backup-container common component: https://developer.gov.bc.ca/Backup-Container 
+
+## How to Create a Mongo Database Backup Cronjob
+
+Note that the template assumes there is a service called ```landuseplanning-api-mongodb``` that is exposes the ```landuseplanning-api-mongodb``` deployment's pod locally. The template also relies on some OpenShift secrets and config maps that will need to be created.
+
+### Through the OpenShift UI or otherwise, you should to create:
+
+OpenShift secret named ```artifactory-creds``` (documentation for artifactory found here: https://developer.gov.bc.ca/Artifact-Repositories-(Artifactory) )
+
+
+OpenShift config map named ```backup-lup-conf``` using the ```backup-configmap.yaml``` file in this directory.
+
+Two PVCs (one for backups, one for verification) using the ```backup-pvc.yaml``` and ```verification-pvc.yaml```
+
+
+### To create the cronjob:
+
+Log in through the UI, and navigate to Workloads->CronJobs.
+
+Create a new CronJob using the ```backup-cron.yaml``` file in this directory, making any changes as required for your environment.
```

Comment:
I don't think so. I wasn't able to get the templates from Backup-container working at all, and I have always had a personal preference for creating templates through the GUI since it allows for a lot more troubleshooting when trying to implement something new like this.

Reply:
if their template doesn't work, we could make our own that does.